packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.mdx

```diff
@@ -0,0 +1,12 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+<Meta title="MDX|ColorPicker" component={ColorPicker} />
```

Comment:
The title describes the location, so in this case it would be `General/ColorPicker`. Component is not needed, since this is a mdx story.
```suggestion
<Meta title="General/ColorPicker"/>
```

Reply:
Ok, yeah I had no idea what to put there and just copied stuff rom another component. Will just omit it next time. :)